.gitlab/pipelines/stages/deploy/deploy_kubecf.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+# shellcheck disable=SC1090
+source "$(bazel info workspace)/.gitlab/pipelines/config/config.sh"
+
+# Locate the built kubecf chart.
+chart="$(find output/ -name 'scf-*.tgz')"
+
+# Create a values.yaml for the test.
+values="$(mktemp -t values_XXXXXXXX.yaml)"
+(cat <<EOF
+system_domain: "${SYSTEM_DOMAIN}"
+
+service:
+  type: ClusterIP
+  clusterIP: "${CLUSTER_IP}"
+
+features:
+  eirini:
+    enabled: ${EIRINI_ENABLED}
+EOF
+) > "${values}"
+
+# Render and apply the kubecf chart.
+bazel run @helm//:helm -- template "${chart}" --name kubecf --namespace "${KUBECF_NAMESPACE}" --values "${values}" | bazel run @kubectl//:kubectl -- apply -f - --namespace "${KUBECF_NAMESPACE}"
```

Comment:
Hmm, we can't just `bazel run //dev/scf:apply` because of the different configs here? Can we do something with `bazel … --define system_domain=${SYSTEM_DOMAIN}`?

Reply:
We could use the `//dev/scf:apply` target here, but we want to exercise the newly built chart, which will then be published.

Reply:
Ah yeah, that makes sense. (See previous confusion about `output/` _not_ being a bazel output).